packages/web-app-files/src/components/FilesList/QuickActions.vue

```diff
@@ -39,3 +39,12 @@ export default {
   }
 }
 </script>
+
+<style lang="scss">
+.quick-action-button {
+  padding: 6px !important;
```

Comment:
Why the hardcoded pixel value for the padding? You can do something like `calc(1.5 * var(--oc-space-xsmall))`.

Reply:
yes that better 👍🏼  couldnt think of this solution